tests/requires-fail/test.rules

```diff
@@ -14,4 +14,4 @@ alert udp any any -> any any (vxlan_vni:10; requires: version >= 10; sid:2;)
 alert http any any => any any (requires: version >= 10; sid:3;)
 alert tcp any any -> any any (frame:smtp.not_supported; requires: version >= 10; sid:4;)
 
-alert asdf any any -> any any (requires: version >= 6, foo bar; sid:102; rev:1;)
+alert asdf any any -> any any (requires: version >= 6; foo: bar; sid:102; rev:1;)
```

Comment:
Why do we change this existing test ?

Reply:
We want the test to fail parsing with an invalid requires expression, not just an unknown one.

Reply:
More details. In master, `requires: version >= 6, foo bar` is considered as all requirements met. So this rule fails out as `asdf` is unknown.  With the PR, this rule is considered without requirements met, so the `asdf` is never parsed. This is to test somewhat of an edge case documented in ticket https://redmine.openinfosecfoundation.org/issues/6710, which I've added to the README now as well.

Reply:
But in the new version `foo: bar;` is totally unused then, right ?

And it would still be good to test `requires: version >= 6, foo bar;` somehow, right ?